localization/english/culture/wc_regional_innovations_l_english.yml

```diff
@@ -0,0 +1,28 @@
+﻿l_english:
+ 
+ ##############
+ # Tribal Era #
+ ##############
+ 
+ innovation_kyparite_sap:0 "Kyparite Sap"
+ innovation_kyparite_sap_desc:0 "Kyparite is the lifeblood of Azeroth. It can heal our wounds and make us stronger. It will be a great prestige to have Kyparite as part of our society."
+ 
+ ######################
+ # Early Medieval Era #
+ ###################### 
+ 
+ innovation_taiga_warfare:0 "Taiga Warfare"
+ innovation_taiga_warfare_desc:0 "Our people have lived long in the freezing taiga, and thus we have become accustomed to its harsh conditions. The cold may only be bitter to our foes."
+ 
+ innovation_desert_warfare:0 "Desert Warfare"
+ innovation_desert_warfare_desc:0 "Our people have lived long in the dry and hot deserts, and thus we have become accustomed to its harsh conditions. As the sands flow, so do we."
+ 
+ innovation_jungle_warfare:0 "Jungle Warfare"
```

Comment:
It's better to pick another name since it's not only about jungles.

Reply:
Ok guerilla warfare sounds good?

Reply:
> Ok guerilla warfare sounds good?

Yes.